datadog_checks_dev/datadog_checks/dev/tooling/commands/env/start.py

```diff
@@ -114,6 +114,9 @@ def start(ctx, check, env, agent, python, dev, base, env_vars, org_name, profile
         profile_memory = False
         echo_warning('No API key is set; collecting metrics about memory usage will be disabled.')
 
+    if not (ctx.obj['repo_choice'] == 'core' and ctx.obj.get('repo') == 'core' and dev):
```

Comment:
```suggestion
    if not dev and ctx.obj['repo_choice'] != 'core':
```

Reply:
@ofek do you not need to check `ctx.obj.get('repo') == 'core'`?